library.properties

```diff
@@ -1,5 +1,5 @@
 name=ArduinoIoTCloud
-version=1.3.1
+version=1.4.0
```

Comment:
Do you bump the library version number in this PR? It's of course possible, but the/my ususal flow is to merge the PR, than make a dedicated "release" commit.

Reply:
ups, this was due to a wrong rebase, 1.4.0 is already released